gulp/typescript.js

```diff
@@ -53,4 +55,17 @@ module.exports = (gulp, plugins, blueprint) => {
             tsResult.dts,
         ]).pipe(blueprint.dest(project));
     });
+
+    const bundleTaskNames = blueprint.projectsWithBlock("typescript").map((project) => {
+        const taskName = `typescript-bundle-${project.id}`;
```

Comment:
I don't mind spelling it out by hand rather than using `blueprint.task()`; I just think the task name is wrong and it shouldn't be in the `gulp/typescript.js` file